common/src/main/kotlin/io/homeassistant/companion/android/common/data/websocket/impl/WebSocketCoreImpl.kt

```diff
@@ -67,17 +69,33 @@ import okio.ByteString
 import okio.ByteString.Companion.toByteString
 import timber.log.Timber
 
+private const val WEBSOCKET_PATH = "api/websocket"
+
+private fun URL.toWebSocketURL(): String {
```

Comment:
I had this change for a while but I can remove it from this PR.

Reply:
I'm going to remove this and keep previous impl. It's a bad idea to use URL here since it doesn't support ws protocol. This impl is also bad because it forgets the host and port.